blackbird_python/blackbird/listener.py

```diff
@@ -177,6 +184,44 @@ def exitTarget(self, ctx: blackbirdParser.TargetContext):
 
         self._program._target["options"] = kwargs
 
+    def exitInclude(self, ctx: blackbirdParser.IncludeContext):
+        """Run after exiting include statement.
+
+        Args:
+            ctx: IncludeContext
+        """
+        filename = os.path.join(self._cwd, ctx.STR().getText()[1:-1])
+
+        # check if filename has already been included
+        for _, f in self._includes.items():
+            if filename == f[0]:
```

Comment:
```suggestion
            if f[0] == filename:
```
(functionally meaningless change, but intent reads better)